app/build.gradle

```diff
@@ -196,6 +196,7 @@ dependencies {
 
     implementation "com.google.android.material:material:1.5.0"
     implementation 'androidx.appcompat:appcompat:1.4.1'
+    implementation "androidx.collection:collection-ktx:1.2.0"
```

Comment:
It doesn't seem like we need to declare this dependency explicitly here.

Reply:
In the older version, the return type of `valueAt` was marked as nullable.